projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.ts

```diff
@@ -180,11 +189,12 @@ export class ItemTransactionComponent implements OnInit, OnDestroy {
         pickupLocationPid,
         this.itemPid
       )
-      .subscribe(() => {
+      .subscribe((data) => {
```

Comment:
data could be typed, at least with "any".

Reply:
Done